packages/appkit/src/universal-adapter/client.ts

```diff
@@ -414,6 +414,10 @@ export class UniversalAdapterClient {
       }
     }
 
+    if (this.appKit?.getCaipAddress() && !this.appKit?.getIsConnectedState()) {
+      this.appKit?.setIsConnected(true, this.appKit?.getActiveChainNamespace() || 'eip155')
+    }
```

Comment:
I wish i could put this logic in a handler, but we're [syncing account](https://github.com/reown-com/appkit/blob/main/packages/appkit/src/universal-adapter/client.ts#L417) before listening to our [handlers](https://github.com/reown-com/appkit/blob/main/packages/appkit/src/universal-adapter/client.ts#L418). 

I tried to refactor it so it's the other way around, but that introduces more issues.